docker-compose.yml

```diff
@@ -244,8 +244,8 @@ services:
     volumes:
       - .:/app:z
     ports:
-      - '9394:9394'
-    command: bundle exec prometheus_exporter -b 0.0.0.0 -t 50 --verbose -a lib/prometheus/graphql_collector.rb -a lib/prometheus/business_collector.rb -a lib/prometheus/engineering_collector.rb
+      - '10000:10000'
```

Comment:
Is it possible to to keep the standard port of 9394?

Reply:
I don't see why not :) It just has to match what we put in devel.json.